staff_features/jobs/step_definitions/jobs.rb

```diff
@@ -15,8 +15,17 @@
 end
 
 Then 'the job completes' do
-  expect(page).not_to have_text 'This job is next in the queue.'
-  expect(page).to have_text 'The job has completed.'
+  tries = 0
+
+  begin
```

Comment:
how about using a "using_wait_time" capybara block instead?

https://www.rubydoc.info/gems/capybara/Capybara/Session#using_wait_time-instance_method

We are using this also in our feature specs: 
https://github.com/archivesspace/archivesspace/pull/3353/files#diff-14c815058850c6270526a9cda66fb6c37dd3e56736a78992e0f0cb56fbe95bcaR218